.circleci/config.yml

```diff
@@ -206,23 +207,28 @@ jobs:
             - run:
                 name: wait for federation demo to start
                 command: npx wait-on tcp:4001 tcp:4002 tcp:4003 tcp:4004 tcp:4000
-
+      - when:
+          condition:
+            not:
+              # TODO [igni]: figure out how to make this work on windows
+              equal: [*rust_windows_executor, << parameters.platform >>]
+          steps:
+            - run:
+                command: >
+                  cargo xtask check
       - run:
           command: >
-            cargo xtask test
+            cargo xtask lint
```

Comment:
check, lint.... :thinking: should it really be 2 different commands?

I think we should keep `lint` and move your check there. You can exclude windows from the test there too using `#[cfg(not(windows))]`.

Check would be fine too if it didn't collide with "cargo check" which means checking if it compiles, not linting. (counter intuitive for rust developers)

Reply:
It can indeed be a bit confusing! Let's try to think of a name that conveys we're making sure the codebase complies to our standards, license and fmt and stylewise.

wdyt?

Reply:
Looking at #147 and this, i think this calls for a team discussion about what xtask does and what the CI does. I'm opening an issue or maybe a discussion ?

Reply:
just opened #160 so we dont forget about it!

Reply:
Purrfect :smile_cat: